### ISSUE TITLE
Totem key does not work in languages other than English

### ISSUE THREAD

**Issue by spdish854** [glitchifyed/quick_hotkeys#9]
If you set the language settings to "English (US), the totem hotkey works fine

but if you set it to another language (I've tried using Korean and LOCAT languages), only the failure sound is output and the totem hotkey doesn't work :(

**Comment by glitchifyed** [comment]
Yes, that's because it scans for any items that contain the word Totem within their item names. This was a change made for other totems from other mods to be swapped in unison. I may include an exception for regular totems via item ids instead in the future.

**Comment by glitchifyed** [comment]
Okay so I've just quickly set it to first search for the totem item directly and if it doesn't find that it will search the item id itself rather than the name of the item. That way no matter what language it is it should still work just as well. Will upload v1.1.3 soon.
![image](https://github.com/user-attachments/assets/61614883-34f9-4f75-a8c6-658682ecef13)
